frontend/app/controllers/skill-search.js

```diff
@@ -19,13 +26,42 @@ export default class SkillSearchController extends Controller {
   @computed("model")
   get selectedSkill() {
     const skillId = this.router.currentRoute.queryParams.skill_id;
+    this.currentSkillId = skillId;
     return skillId ? this.get("store").peekRecord("skill", skillId) : null;
   }
 
+  @action
+  updateSelection() {
+    this.get("router").transitionTo({
+      queryParams: {
+        skill_id: this.currentSkillId,
+        level: this.get("levelValue")
+      }
+    });
+  }
+
+  get levelName() {
+    const levelNames = [
+      "Nicht bewertet",
+      "Trainee",
+      "Junior",
+      "Professional",
+      "Senior",
+      "Expert"
+    ];
```

Comment:
Refactor this into the people-skills.js model 
Just like here: https://github.com/puzzle/skills/blob/d54cb8202395bd6203d834601f98cef8fee04964/frontend/app/models/person.js#L33

Reply:
Did you mean people-skill.js that is in the same folder? people-skills.js doesn't exist :P

Reply:
right, I meant exactly that ;)